.github/workflows/ci.yml

```diff
@@ -642,6 +642,11 @@ jobs:
       env:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
         DD_APPLICATION_KEY: ${{ secrets.DD_APPLICATION_KEY }}
+        DD_APP_KEY: ${{ secrets.DD_APPLICATION_KEY }}
```

Comment:
could you move this env setto only your scenario? (its better to not expose them too much).